README.md

```diff
@@ -1 +1,205 @@
-# to-do
```

Comment:
**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍